src/cmap/connection.ts

```diff
@@ -304,28 +361,34 @@ function streamIdentifier(stream: any) {
 }
 
 // Not meant to be called directly, the wire protocol methods call this assuming it is a `Pool` instance
-function write(this: any, command: any, options: any, callback: Function) {
+function write(
+  this: Connection,
+  command: WriteProtocolMessageType,
+  options: CommandOptions,
+  callback: Callback
+) {
   const connection = this;
   if (typeof options === 'function') {
     callback = options;
   }
 
   options = options || {};
-  const operationDescription = {
+  const operationDescription: OperationDescription = {
     requestId: command.requestId,
     cb: callback,
     session: options.session,
-    fullResult: typeof options.fullResult === 'boolean' ? options.fullResult : false,
-    noResponse: typeof options.noResponse === 'boolean' ? options.noResponse : false,
+    fullResult: 'boolean' === typeof options.fullResult ? options.fullResult : false,
```

Comment:
This looks like just a "whitespace" change, can we make these changes in subsequent PRs if we all feel like the pattern helps readability?